test/e2e/managed_test.go

```diff
@@ -61,3 +61,14 @@ func TestEditManagedRoute(t *testing.T) {
 		t.Fatalf("error: %s", err)
 	}
 }
+
+func TestEditBrandingFromManagedConfigMap(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
+	defer testframework.MustManageConsole(t, client)
+	testframework.MustManageConsole(t, client)
+
+	err := mergeAndCheckConfigMap(t, client, true)
```

Comment:
So, your e2e test shouldn't really know about the merging, that happens in the `operator`.  This func really just needs to:

- `GET` `configmap` `console-config` in `openshift-config-managed`,  set `brand=something`, and `UPDATE/PUT` it back. 
- Wait a second (but the `UPDATE/PUT` is blocking, so should be fine)
- `GET` the `configmap` `console-config` in `openshift-console` and see if it has `brand=something`